src/status_im2/common/data_store/wallet.cljs

```diff
@@ -1,11 +1,50 @@
 (ns status-im2.common.data-store.wallet
   (:require
-    clojure.set))
+    [clojure.set :as set]
+    [clojure.string :as string]
+    [status-im2.constants :as constants]))
+
+(defn chain-ids-string->set
+  [ids-string]
+  (->> (string/split ids-string constants/chain-id-separator)
+       (map js/parseInt)
```

Comment:
we have a `utils.number/parse-int` function, it's helpful because we can set a fallback value.

just a small side note, we can only use `into`:
```clojure
(into #{}
      (map js/parseInt)
      (string/split ,,,))
```